src/resolvers/subjectResolvers.ts

```diff
@@ -19,12 +19,12 @@ export const Query = {
   ): Promise<GQLSubject> {
     const list = await fetchSubjects(context);
     return list
-      .filter(s => s.metadata.visible === true)
+      .filter(s => (s.metadata && s.metadata.visible) || true)
```

Comment:
Vil ikke dette alltid bli true så `visible: false` ikke blir filtrert ut?
```suggestion
      .filter(s => !(s?.metadata?.visible === false))
```

Reply:
Her vil vel den også returnere true dersom metadata er null/undefined. 
Dersom det er ønskelig å inkludere tilfeller der metadata ikke er tilstede er det greit, men om det ikke er ønskelig så syntes jeg ikke vi burde det.